src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/OpenDistroSecurityConfigAction.java

```diff
@@ -58,21 +63,18 @@ public OpenDistroSecurityConfigAction(final Settings settings, final Path config
     }
 
     @Override
-    protected void registerHandlers(RestController controller, Settings settings) {
-        controller.registerHandler(Method.GET, "/_opendistro/_security/api/securityconfig/", this);
-
-        //controller.registerHandler(Method.GET, "/_opendistro/_security/api/config/", this);
+    public List<Route> routes() {
+        List<Route> routes = new ArrayList<>();
```

Comment:
Can static `routes` be used as there is already `allowPutOrPatch` flag that filters not implemented requests.